app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      respond_to :json
+
+      def index
+        @filter_by = Book.ransack(params[:filter_by])
```

Comment:
you changed the variable name but the name still does not indicate on the value held by the variable it may better be stored as filter_book maybe ?